imgui.go

```diff
@@ -638,3 +638,21 @@ func IsItemHoveredV(flags int) bool {
 func IsItemHovered() bool {
 	return IsItemHoveredV(HoveredFlagsDefault)
 }
+
+// ListBox creates a imgui::Listbox
+func ListBox(label string, currentItem *int32, items []string, itemsCount int, heightItems int) bool {
```

Comment:
Furthermore, `heightItems` is an optional parameter. As per naming convention (see README/CONTRIBUTING), please extend this to be `ListBoxV(string, *int32, []string, int)` and `ListBox(string, *int32, []string)`. The second with corresponding comment of defaults (see others, as well as in CONTRIBUTING).